simple/datasets/email/training_data.py

```diff
@@ -265,4 +265,276 @@
             "deps": ["ROOT", "-", "NOUN", "-", "TO"],
         },
     ),
+
```

Comment:
reminder data should not be part of the email dataset. remove it or move it to reminder.